dance/tests/dance_pipeline_test.py

```diff
@@ -43,6 +42,20 @@ def _assert_smiles_in_oeb_are_equal(oeb: pathlib.Path, smiles: List[str]):
     assert set(smiles) == outputted_smiles
 
 
+@pytest.fixture
```

Comment:
A fixture is just something that can be used by multiple tests.This fixture provides several useful temporary files.

Reply:
See line 163-164 for an example of how this fixture is used.